pyro/nn/module.py

```diff
@@ -14,9 +14,10 @@
 """
 import functools
 import inspect
+from typing import Union
 import weakref
 from collections import OrderedDict, namedtuple
-
+from torch._jit_internal import _copy_to_script_wrapper
```

Comment:
I'm a little worried about depending on an internal function here. What do you think of limiting damage scope by guarding this import into a try-catch block:
```py
import warnings

try:
    from torch._jit_internal import _copy_to_script_wrapper
except ImportError:
    warnings.warn("Cannot find torch._jit_internal._copy_to_script_wrapper", ImportWarning)

    # Fall back to trivial decorator.
    def _copy_to_script_wrapper(fn):
        return fn
```

Reply:
that is good foresight, I added that in fae8ba6

Reply:
that is good foresight, fixed it via 403f0d4